.github/workflows/ci_bump_libkernel.yml

```diff
@@ -0,0 +1,37 @@
+name: "CI - Bump libkernel"
+
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: "Semver type of new version (major / minor / patch)"
+        # Input has to be provided for the workflow to run
+        required: true
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+
+defaults:
+  run:
+    working-directory: "libs/libkernel"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
this isn't needed because the only trigger for this workflow is the `workflow_dispatch`